docs/user/operation-guides/02-30-enable-istio-access-logs.md

```diff
@@ -24,12 +24,22 @@ The [log format](https://github.com/kyma-project/istio/blob/main/manifests/istio
 
 ## Configuration
 
-Use the Telemetry API to selectively enable Istio access logs. You can enable access logs for an entire namespace, for a selective workload, or on the Istio Gateway scope.
+Use the Telemetry API to selectively enable Istio access logs. See:
+- [Configure Istio Access Logs for the Entire Namespace](#configure-istio-access-logs-for-the-entire-namespace)
+- [Configure Istio Access Logs for a Selective Workload](#configure-istio-access-logs-for-a-selective-workload)
+- [Configure Istio Access Logs for a Specific Gateway](#configure-istio-access-logs-for-a-specific-gateway)
+- [Configure Istio Access Logs for the Entire Mesh](#configure-istio-access-logs-for-the-entire-mesh)
 
 ### Configure Istio Access Logs for the Entire Namespace
 
-1. In the following sample configuration, replace `{YOUR_NAMESPACE}` with your namespace.
-1. To apply the configuration, run `kubectl apply`.
+<!-- tabs:start -->
+
+#### **Kyma Dashboard**
+
+1. Go to the namespace for which you want to configure Istio access logs.
+2. Go to **Configuration > Custom Resources**.
+3. Scroll down to find the section `telemetry.istio.io` and select **Telemetries**.
```

Comment:
```suggestion
3. In the section `telemetry.istio.io`, select **Telemetries**.
```
I would avoid mentioning where exactly it is found (somewhere low, visible only after scrolling); otherwise you'll have to update the instructions if the UI changes so that scrolling down is no longer needed.